frontend/src/pages/application/Pipeline/create.vue

```diff
@@ -1,26 +1,12 @@
 <template>
-    <Teleport
```

Comment:
Think I'm going mad, but can't work out (even though I can see it works) how we got away with deleting all of these Teleports?

Reply:
discussed ona c all, was code not doing anything, not sure how it got here in the first place.